src/resources/UsageAnalytics/Read/DataHealth/DataHealthInterfaces.ts

```diff
@@ -310,3 +310,57 @@ export interface DataHealthGetTrackingIdsResponse {
      */
     values: DataHealthFacetValue[];
 }
+
+export interface DataHealthGetFailedInstancesParams extends OrganizationParamParts, TimeRangeParamParts {
+    /**
+     * The data health criterion for which the failed instances should be returned.
+     */
+    criterionId: string;
+    /**
+     * The group for which data health information should be returned.
+     */
+    group: string;
+    /**
+     * The value of the criterion scope for which failed instances should be returned.
+     */
+    scopeValue: string;
+    /**
+     * The tracking IDs for which the results should be returned.
+     */
+    trackingId: string[];
+}
+
+export interface DataHealthGetFailedInstancesResponse extends PaginatedResponse {
+    /**
+     * A collection of failed instances.
+     */
+    failedInstances: DataHealthFailedInstanceEntry[];
+}
+
+export interface DataHealthFailedInstanceEntry {
```

Comment:
The response schema isn't documented in https://platformdev.cloud.coveo.com/docs?urls.primaryName=Usage%20Analytics%20Read#/Data%20Health%20API%20-%20Version%2015/get__v15_datahealth_criteria_failedInstances

Reply:
We should still be able to add it to the platform-client while documenting it in the data health API in the meantime, no?

Reply:
@fbeaudoincoveo is this blocking or no?

Reply:
It's going to be added with https://coveord.atlassian.net/browse/UA-9392

Reply:
Not blocking, no